packages/ripple-tide-grant/definition.yaml

```diff
@@ -0,0 +1,41 @@
+type: object
+title: Grant
+description: Tide mappings for Grant content type
+required:
+  - title
+  - summary
+  - bodyComponents
+  - headerComponents
+allOf:
+  - $ref: '#/components/schemas/TidePage'
+properties:
+  type:
+    type: string
+    enum:
+      - grant
+  title:
```

Comment:
Will need to add the fields you've added to this schema too. See https://json-schema.org/understanding-json-schema/index.html for syntax. I'd like to explore how we might be able to move this to an annotation based setup instead of having to do this eg: https://github.com/john-doherty/jsdoc-to-json-schema#input-singleton